### DISCUSSION TITLE
Infinite duplication of static image of item moving around

### DISCUSSION THREAD

**Discussion by angelalaw01887** [tom-englert/RegionToShare#22] [Q&A] [answered]
**Describe the bug**
Say I move the cursor arrow around the region that is shared.  I will see it trail of every spot it logged itself at.  So I could move it around in the region that is sharing on teams and I can see in the region area 40 static cursor image every spot it registered.

**To Reproduce**
Windows 10 Pro/Select a region to share and run it
Open Microsoft teams/chat/Share Screen/Share Windows/Region Share
Goto region area move cursor around and notice it is just replicating the cursor image everywhere it has moved and the image of the cursor doesn't go away.  Same goes for windows.

**Expected behavior**
To not have replication happen of items that are being moved around by the mouse or the mouse itself

**Comment by tom-englert** [comment, marked as answer]
Just clicking into the region window should remove this.

**Comment by angelalaw01887** [comment]
@tom-englert  thanks so much that worked like a champ works well with display fusion create the zones....
